supplementary/supplementary_tests/src/test/test_alica_dynamicloadcreator.cpp

```diff
@@ -231,5 +181,36 @@ TEST_F(AlicaDynamicLoading, simple_waitbehaviour_load)
     ASSERT_EQ("waitbehaviour", behaviour->getName());
 }
 
+TEST_F(AlicaDynamicLoading, simple_behaviour_fail_load_wrong_path)
+{
+    std::string path = getRootPath();
+
+    YAML::Node node;
+    try {
+        node = YAML::LoadFile(path + "/etc/plans/behaviours/Acme.beh");
+    } catch (YAML::BadFile& badFile) {
+        Logging::logError("DT") << path + "/etc/plans/behaviours/Acme.beh";
+        AlicaEngine::abort("MM: Could not parse behaviour file: ", badFile.msg);
+    }
+
+    // Load model
+    Behaviour* behaviourModel;
+    behaviourModel = BehaviourFactory::create(node);
+
+    // Export wrong path
+    std::string wrongPath = "LD_LIBRARY_PATH=/wrong_path/";
+    char* env = &wrongPath[0];
+    putenv(env);
```

Comment:
I assume LD_LIBRARY_PATH is set in the env by github action right?
So won't putenv() change this value? If so we are relying on the fact that this test should execute last otherwise other tests will also fail right?

Reply:
Maybe better to just set this up as a unit test instead of actually trying to load it and change the env variable

Reply:
I think we can remove the negative test that currently exists, and instead implement a test of trying to load a behavior or plan that doesn't exist (even though LD_LIBRARY_PATH does contain some plan libraries.  Or just remove.  I don't like having tests that mess with external env variables etc anyway.